frontend/src/views/material/MaterialLists.vue

```diff
@@ -0,0 +1,50 @@
+<!--
+Show all material lists for a camp on mobile
+-->
+
+<template>
+  <content-card :title="$tc('views.material.materialLists.title')" toolbar>
+    <template v-if="!isGuest" #title-actions>
+      <DialogMaterialListCreate :camp="camp()">
+        <template #activator="{ on }">
+          <ButtonAdd class="mr-n2" height="32" v-on="on"
+            >{{ $tc('global.button.create') }}
+          </ButtonAdd>
+        </template>
+      </DialogMaterialListCreate>
+    </template>
```

Comment:
So you actively decided to keep this button in this view?

Reply:
I think this makes sense